code/modules/surgery/bodyparts/parts.dm

```diff
@@ -67,7 +67,7 @@
 	is_dimorphic = FALSE
 	wound_resistance = -10
 	bodytype = BODYTYPE_MONKEY | BODYTYPE_ORGANIC
-	acceptable_bodytype = BODYTYPE_HUMANOID
+	acceptable_bodytype = BODYTYPE_MONKEY
```

Comment:
im surprised monkey chest bodytype didnt horrifically break the game in other ways. I probably made "special" limb attachment like species changing ignore the bodytype check